Userland/Libraries/LibWeb/DOMTreeModel.cpp

```diff
@@ -130,10 +132,15 @@ GUI::Variant DOMTreeModel::data(const GUI::ModelIndex& index, GUI::ModelRole rol
     }
 
     if (role == GUI::ModelRole::Icon) {
-        if (type == "document")
+        if (type == "document"sv)
             return m_document_icon;
-        if (type == "element")
+        if (type == "element"sv) {
+            if (node_name == "SCRIPT"sv)
+                return m_script_icon;
+            if (node_name == "STYLE"sv)
```

Comment:
This check, as far as I can tell, means that it doesn't work on <link> external CSS.

Reply:
You are totally right. I'll fix it now.

Reply:
Fixed.